src/components/UI/app_container.js

```diff
@@ -0,0 +1,153 @@
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from '@emotion/styled';
+
+const Wrapper = styled.div(props => ({
+  width: 'calc(25% - 15px)',
+  flexShrink: '0',
+  margin: '0 20px 20px 0',
+  '&:nth-of-type(4n + 0)': {
+    marginRight: '0px',
+  },
+
+  [`@media only screen and (max-width: ${props.theme.breakpoints.large})`]: {
+    width: 'calc(33.333% - 13.333px)',
+    // we are unsetting the 4n + 0 rule one above
```

Comment:
It's a bit nit picky, and I used to not care about this, but I'd like join the Golang crew in the comment style, which is full sentence. So starting with a capital letter and ending with a dot.

```suggestion
    // We are unsetting the 4n + 0 rule one above.
```